openapi/generated/errorHelpers.ts

```diff
@@ -7,15 +8,17 @@ import { AxiosError } from "axios";
  * @returns true if error originated from the API
  */
 export const isServiceApiError = (error: unknown): error is AxiosError => {
-    return (error as AxiosError).response?.data.code !== undefined;
+    const errorListResponse = (error as AxiosError).response?.data as ErrorListResponse;
+    return errorListResponse?.items !== undefined;
 };
 
 /**
- * Get the error code from the API error
+ * Get the error code related to the first error item returned from the API
  *
  * @param error generic error returned from function
  * @returns error code (one of fields of APIErrorCodes)
  */
 export const getErrorCode = (error: unknown): string | undefined => {
-    return (error as AxiosError).response?.data?.code;
+    const errorListResponse = (error as AxiosError).response?.data as ErrorListResponse;
+    return errorListResponse?.items?.length ? errorListResponse?.items[0].code : undefined;
```

Comment:
as this is part of `openapi/generated` folder, won't we lost this change with every sdk re-generation?

Reply:
Only `model` and `api` folders are going to be removed during (re)generation: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/2a3e1408376ea5847113088eabef61889332536c/scripts/openapi-sdk/generate_openapi_client.sh#L20